src/vs/vscode.d.ts

```diff
@@ -5447,14 +5452,17 @@ declare module 'vscode' {
 	export namespace scm {
 
 		/**
-		 * The [input box](#SourceControlInputBox) in the Source Control viewlet.
+		 * The [input box](#SourceControlInputBox) for the last source control
+		 * created by the extension.
+		 *
+		 * @deprecated Use [SourceControl.inputBox](#SourceControl.inputBox) instead
 		 */
 		export const inputBox: SourceControlInputBox;
 
 		/**
 		 * Creates a new [source control](#SourceControl) instance.
 		 *
-		 * @param id A unique `id` for the source control. Something short, eg: `git`.
+		 * @param id An `id` for the source control. Something short, eg: `git`.
```

Comment:
Should we rename this to `type` or context key?

Reply:
That would make sense. Should I then create a `type` field in `SourceControl` and deprecate `id`?

Reply:
Not going with this proposal.